ha/setup/ha_setup.py

```diff
@@ -1086,10 +1086,11 @@ def main(argv: list):
     try:
         if sys.argv[1] == "post_install":
             Conf.init()
-            Conf.load(const.HA_GLOBAL_INDEX, f"yaml://{const.SOURCE_CONFIG_FILE}")
```

Comment:
FYI: this file is not used for k8s mini provisioning 
Check if similar change needs to be done in k8s_setup/ha_setup.py

Reply:
I removed all log.init and moved in comfigmanager

I think we should remove not needed file or change file if changing common utility

Let me know if needed i can revert this

Reply:
This change is fine. 
Here I see that conf.load is replaced with ConfigManager._safe_load
Same thing should be done in k8s_setup/ha_setup.py also

Reply:
conclusion from disussion, conf,load in k8s_setup loads from _url which is cluster.conf
Current change there is fine